src/DaylightSavingTime.cpp

```diff
@@ -67,9 +67,11 @@ void DaylightSavingTime::determineDstStartAndEnd(const tm &givenTm)
     tm dstEndTm = givenTm;
     if (DST_LOCATION == Europe)
     {
+        int offset_minutes = static_cast<int>(UTC_OFFSET * 60);
+
         // In Europe, DST starts on last Sunday in March at 01:00 UTC
         dstStartTm.tm_sec = 0;
-        dstStartTm.tm_min = UTC_OFFSET * 60 % 60;
+        dstStartTm.tm_min = (offset_minutes % 60 + 60) % 60;
```

Comment:
Can this be simplified as following?
```suggestion
        dstStartTm.tm_min = offset_minutes % 60;
```

Reply:
Yes, because the whole project use camel case (e.g. dstEndTm) rather than snake case (e.g. dst_end_tm) for local variables.

Reply:
Do you have a local repo that you can change accordingly and push to your https://github.com/eeyone/PicoClockGreenEasy/tree/main fork? If not, let me know. Since only a few lines have been changed, I can do the same changes directly in the project repo.